health/job.py

```diff
@@ -92,6 +92,17 @@ def _get_driver(driver_type):
         raise
 
 
+def ignore_exceptions(func):
```

Comment:
we should use https://docs.python.org/2/library/functools.html#functools.wraps here I will add it later